README.md

```diff
@@ -1,3 +1,26 @@
 # Colin
 
-Tool to check generic rules/best-practices for containers/images/dockerfiles, not a framework for making custom container tests.
+Tool to check generic rules/best-practices for containers/images/dockerfiles.
+
+Initial plan is to validate containers/images/dockerfiles against different ecosystems:
+ - Red Hat Container Catalogue
+ - Fedora Infra (and container guidelines)
+ - CentOS
+ - Atomic Container Best Practices
+
+*Colin* will also provide generic checks for maintainers or users of containerized content,
+
+
+## Technical details
+
+*Colin* will be available as a Python API, and the command line interface.
+
+Each ecosystem defines set of checks for validation and their severity.
```

Comment:
*Each ecosystem will define a set of checks to validate the artifacts. Checks will have different severity level so that we can classify checks as required or optional.*